### ISSUE TITLE
Autosubmit database (SQLite) persistence is not working

### ISSUE THREAD

**Issue by kinow** [BSC-ES/autosubmit#241]
In GitLab by @ jlope2 on Feb 20, 2017, 18:39

##### Steps to reproduce the bug:

* Set the following configuration in the ```autosubmit_xxxx.conf``` file:

```
[storage]
TYPE = db
```
* Run the experiment:

```
$> autosubmit create xxxx
Preparing .lock file to avoid multiple instances with same expid.

Checking configuration files...
autosubmit_a0fl.conf OK
platforms_a0fl.conf OK
jobs_a0fl.conf OK
expdef_a0fl.conf  OK
Configuration files OK

Loading parameters...

Creating the jobs list...
Creating jobs...
Adding dependencies...
Removing redundant dependencies...

Saving the jobs list...
[CRITICAL] Unhandled exception on Autosubmit: table job_list has 11 columns but 12 values were supplied
Traceback (most recent call last):
  File "/home/Earth/jlope2/Development/PycharmProjects/autosubmit/autosubmit/autosubmit.py", line 342, in parse_args
    return Autosubmit.create(args.expid, args.noplot, args.hide, args.output)
  File "/home/Earth/jlope2/Development/PycharmProjects/autosubmit/autosubmit/autosubmit.py", line 1673, in create
    job_list.save()
  File "/home/Earth/jlope2/Development/PycharmProjects/autosubmit/autosubmit/job/job_list.py", line 516, in save
    self._persistence.save(self._persistence_path, self._persistence_file, self._job_list)
  File "/home/Earth/jlope2/Development/PycharmProjects/autosubmit/autosubmit/job/job_list_persistence.py", line 137, in save
    self.db_manager.insertMany(self.JOB_LIST_TABLE, jobs_data)
  File "/home/Earth/jlope2/Development/PycharmProjects/autosubmit/autosubmit/database/db_manager.py", line 90, in insertMany
    cursor.executemany(insert_many_command, data)
OperationalError: table job_list has 11 columns but 12 values were supplied
```
##### The observed behaviour 
There's a Python crash while creating the experiment.

##### The expected behaviour is
```create```progress should work normally.

**Comment by kinow** [comment]
In GitLab by @ jlope2 on Feb 27, 2017, 15:18

Mentioned in commit b6de48044924827f8f19dfeb836d8d0c5f0a3f17

**Comment by kinow** [comment]
In GitLab by @ jlope2 on Feb 27, 2017, 15:29

Mentioned in commit 90de81d541a1af6ce230e88c63c42a1697d7a08a

**Comment by kinow** [comment]
In GitLab by @ jlope2 on Feb 27, 2017, 18:16

Fixed! 

Available since Autosubmit v3.7.8.

**Comment by kinow** [comment]
In GitLab by @ jlope2 on Mar 1, 2017, 11:55

Mentioned in commit f39901aad19b4e4b79768a9135ce8092606e84d1
